internal/pkg/cli/app_package.go

```diff
@@ -112,25 +126,40 @@ func (o *packageAppOpts) Ask() error {
 
 // Execute prints the CloudFormation template of the application for the environment.
 func (o *packageAppOpts) Execute() error {
+	if o.OutputDir != "" {
```

Comment:
I think we had this if-block after `o.store.GetEnvironment(o.ProjectName(), o.EnvName)` on purpose so that if the `GetEnvironment` request fails then we don't end up creating files for no reason.